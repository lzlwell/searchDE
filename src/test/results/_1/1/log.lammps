LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26726 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -921.55064   -921.55064      17.1317           10           50 
       1   -1053.0382   -1053.0382      17.1317           10           50 
       2   -1103.8567   -1103.8567      17.1317           10           50 
       3   -1133.5194   -1133.5194      17.1317           10           50 
       4   -1164.5506   -1164.5506      17.1317           10           50 
       5   -1177.1083   -1177.1083      17.1317           10           50 
       6   -1185.4349   -1185.4349      17.1317           10           50 
       7    -1192.514    -1192.514      17.1317           10           50 
       8   -1193.6363   -1193.6363      17.1317           10           50 
Loop time of 0.31936 on 1 procs for 8 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -921.550639247     -1192.51396786     -1193.63627431
  Force two-norm initial, final = 1318.66 25.049
  Force max component initial, final = 694.222 13.7111
  Final line search alpha, max atom move = 0.0032363 0.0443732
  Iterations, force evaluations = 8 10

Pair  time (%) = 0.138805 (43.4635)
Neigh time (%) = 0.177393 (55.5463)
Comm  time (%) = 0.00126243 (0.395299)
Outpt time (%) = 0.000106812 (0.0334455)
Other time (%) = 0.00179315 (0.561481)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2409 ave 2409 max 2409 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45804 ave 45804 max 45804 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45804
Ave neighs/atom = 152.68
Neighbor list builds = 8
Dangerous builds = 8
min_modify line quadratic

clear
