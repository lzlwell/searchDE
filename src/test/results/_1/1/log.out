LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26726 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -921.55064   -921.55064      17.1317           10           50 
       1   -1053.0382   -1053.0382      17.1317           10           50 
       2   -1103.8567   -1103.8567      17.1317           10           50 
       3   -1133.5194   -1133.5194      17.1317           10           50 
       4   -1164.5506   -1164.5506      17.1317           10           50 
       5   -1177.1083   -1177.1083      17.1317           10           50 
       6   -1185.4349   -1185.4349      17.1317           10           50 
       7    -1192.514    -1192.514      17.1317           10           50 
       8   -1193.6363   -1193.6363      17.1317           10           50 
Loop time of 0.31936 on 1 procs for 8 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -921.550639247     -1192.51396786     -1193.63627431
  Force two-norm initial, final = 1318.66 25.049
  Force max component initial, final = 694.222 13.7111
  Final line search alpha, max atom move = 0.0032363 0.0443732
  Iterations, force evaluations = 8 10

Pair  time (%) = 0.138805 (43.4635)
Neigh time (%) = 0.177393 (55.5463)
Comm  time (%) = 0.00126243 (0.395299)
Outpt time (%) = 0.000106812 (0.0334455)
Other time (%) = 0.00179315 (0.561481)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2409 ave 2409 max 2409 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45804 ave 45804 max 45804 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45804
Ave neighs/atom = 152.68
Neighbor list builds = 8
Dangerous builds = 8
